sys-utils/lsns.c

```diff
@@ -1080,6 +1150,15 @@ int main(int argc, char *argv[])
 		case 'W':
 			ls.no_wrap = 1;
 			break;
+		case 'T':
+			ls.nstree = LSNS_NSTREE_OWNER;
+			if (optarg) {
```

Comment:
You need:
```C
if (*optarg == '=')
  optarg++;
```
to make it robust, getopt_long() is horrible bastard when use optional arguments :-)

Reply:
Updated my change. Thank you.